src/token_library/src/macros/mintable.nr

```diff
@@ -10,6 +9,8 @@ pub global MINTABLE_CODE: Quoted = quote {
         to: AztecAddress,
         amount: U128,
     ) {
+        OwnableLib::only_owner_private(&mut context);
```

Comment:
not ideal bc we might have a minters list handled by the owner, we should call some `MintLib::validate_minter` and implement MintLib to read owner (for now)